### PR TITLE
Feature/descw 977 linter improvement 2

### PR DESCRIPTION
**Pull request by mhaswell-bcgov**

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "extends": "@wordpress/stylelint-config/scss",
+    "extends": "./node_modules/@bcgov/webpack-wordpress/.stylelintrc",
```
Comment (ShawnTurple) [inline .stylelintrc]: Can you not simply do this 
```suggestion
    "extends": "@bcgov/webpack-wordpress/.stylelintrc",
```

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
   "main": "index.js",
   "devDependencies": {
     "@bcgov/bc-sans": "^1.0.1",
+    "@bcgov/webpack-wordpress": "git+https://apps.itsm.gov.bc.ca/bitbucket/scm/wp/bcgov-webpack-wordpress.git",
```
Comment (ShawnTurple) [inline package.json:23]: It might be a good idea to attach a version specific to the weback-wordpress script

**Review by ShawnTurple — APPROVED** [review]

File: .stylelintrc (hunk 1 above)
Comment (mhaswell-bcgov) [reply .stylelintrc]: You can, it turns out. Good catch.

File: package.json (hunk 2 above)
Comment (mhaswell-bcgov) [reply package.json:23]: Looks like we can add version requirements to the end with a # (https://docs.npmjs.com/cli/v8/configuring-npm/package-json#git-urls-as-dependencies) so I'll see about doing that in the next PR.

**Pull request merged**